po/es.po

```diff
@@ -0,0 +1,173 @@
+# Spanish translations for com.github.ryonakano.pinit package.
+# Copyright (C) 2021 THE com.github.ryonakano.pinit'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the com.github.ryonakano.pinit package.
+# Jeyson Antonio Flores Deras <jeyson.flores@protonmail.com>, 2021.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: com.github.ryonakano.pinit\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-10-01 15:41+0900\n"
+"PO-Revision-Date: 2021-10-04 01:02-0600\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 3.0\n"
+"Last-Translator: Jeyson Antonio Flores Deras <jeyson.flores@protonmail."
+"com>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Language: es\n"
+
+#: src/Views/EditView.vala:27
+msgid "File Name"
+msgstr "Nombre del archivo"
+
+#: src/Views/EditView.vala:29
+msgid ""
+"File name of the desktop file. Only lowercase letters and numbers are "
+"allowed."
+msgstr "Nombre del archivo de escritorio. Solo se aceptan letras y números."
+
+#: src/Views/EditView.vala:45
+msgid "App Name"
+msgstr "Nombre de la aplicación"
+
+#: src/Views/EditView.vala:46
+msgid "Shown in Applications Menu or Dock."
+msgstr "Mostrar en el menú de aplicaciones o en el dock."
+
+#: src/Views/EditView.vala:57
+msgid "Comment"
+msgstr "Comentario"
+
+#: src/Views/EditView.vala:58
+msgid "A tooltip text to describe what the app helps you to do."
+msgstr "Una descripción que diga lo que la aplicación te ayuda a hacer."
+
+#: src/Views/EditView.vala:69
+msgid "Exec File"
+msgstr "Archivo ejecutable"
+
+#: src/Views/EditView.vala:71
+msgid "Location of the app itself in an absolute path or an app's alias name."
+msgstr ""
+"Locación de la aplicación en una ruta absoluta o el alias del nombre de una "
+"aplicación."
+
+#: src/Views/EditView.vala:84
+msgid "Icon File"
+msgstr "Archivo del icono"
+
+#: src/Views/EditView.vala:86
+msgid ""
+"Location of an icon for the app in an absolute path or an icon's alias name."
+msgstr ""
+"Locación de un icono para la aplicación en una ruta absoluta o el alias del "
+"nombre de un icono."
+
+#: src/Views/EditView.vala:99
+msgid "App Categories"
+msgstr "Categorías de aplicación"
+
+#: src/Views/EditView.vala:100
+msgid "Type of the app."
+msgstr "Tipo de aplicación"
```

Comment:
I don't understand Spanish, but are you possibly missing a trailing period here?